src/ENVPAK.CPP

```diff
@@ -322,7 +345,7 @@ signal( SIGFPE, 			// floating point errors, and integer errors (changes 0:0) un
 	//    to ease reporting calling location (in which case aborts, no return). */
 
 //--- save current drive/dir to restore at exit
-	getcwd( cwdSave, _MAX_PATH);	// save current dir (including drive) in static for byebye to use at exit
+	cwdSave = xgetdir();	// save current dir (including drive) in static for byebye to use at exit
```

Comment:
This will not work.  xgetdir returns a pointer into tmpstr which will (probably) not survive the entire run.

What is needed here is 1) leave cwdSave as it was (character array).  2) strcpy( cwdSave, xgetdir()) or words to that effect.

Or we change xgetdir() to take a pointer or return a std::string.

We should perhaps have a short call about tmpstr?

Reply:
I implemented the first option, returning cwdSave to a character array.